src/aap_eda/api/serializers/activation.py

```diff
@@ -111,6 +162,10 @@ def to_representation(self, activation):
             CredentialSerializer(credential).data
             for credential in activation.credentials.all()
         ]
+        event_streams = [
```

Comment:
This would return a list of objects. Do we want this for the list view? Should be just the list of id's?

Reply:
@Alex-Izquierdo The UI needs the object not the ID since it gets displayed.

Reply:
@hsong-rh @lgalis mentioned she just needs the id and name so we can trim the object